xdp-loader/README.org

```diff
@@ -77,6 +77,16 @@ file. The default is to use the first program in each file. Only one of
 --section and --prog-name may be specified. If this option is set, it applies to
 all programs being loaded.
 
+** -P, --prio <priority>
+Specify with BPF program with the run priority to load the XDP program(s) from
+in each file. The default is to use the program metadata of ELF section. If this
```

Comment:
The first sentence reads a bit strangely. Maybe: "Specify the priority to load the XDP program(s) with (this affects the order of programs running on the interface)." 

We should also specify the default if no metadata is set for the program, so the second sentence should be "The default is to use the value from the metadata in the program ELF file, or a value of 50 if the program has no such metadata."